.travis.yml

```diff
@@ -17,14 +17,6 @@ jobs:
           --verbose_test_summary 
           --instrumentation_filter=//... 
           --test_output=errors
-          --experimental_use_llvm_covmap
-          --collect_code_coverage
-        # Convert profdata results to lcov. We have to do this because of the
-        # current state of support for clang compiler coverage results in
-        # bazel. Once https://github.com/bazelbuild/bazel/issues/9406 is 
-        # addressed, we can probably change the `coverage_feature` flags in
-        # `cc_toolchain_config.bzl` to just `--coverage` and remove this script
-        - ./convert_profdata_to_lcov.sh
         # Upload coverage results
         - bash <(curl -s https://codecov.io/bash)
```

Comment:
Looks like codecov isn't picking up any coverage results. I believe we need to change this line to:
``` suggestion
bash <(curl -s https://codecov.io/bash) -s src/bazel-testlogs/ | (head -n100 && tail -n100)
```
and it should work.